instagram/src/components/PostContainer/PostPage.js

```diff
@@ -0,0 +1,33 @@
+import React, { Component } from 'react';
+import dummyData from '../../utilities/dummy_data';
+import PostContainer from './PostContainer';
+import SearchBar from '../SearchBar/SearchBar';
+import Authenticate from '../Authentication/Authenticate';
+
+class PostPage extends Component {
+  constructor() {
+    super();
+    this.state = {
+      posts: dummyData,
+      searchQuery: ''
+    }
+  }
+
+  onSearch(searchQuery) {
+    this.setState({
+      ...this.state,
+      searchQuery
+    })
+  }
+
+  render() {
+    return (
+      <div className="App">
+        <SearchBar logOut={this.props.logOut} onSearch={(searchQuery) => this.onSearch(searchQuery)} />
```

Comment:
Why make this a bit roundabout and have the arrow function here? It'd be easier and arguably better to have the method itself be an arrow func:
```javascript
  onSearch = searchQuery => {
    this.setState({
      ...this.state,
      searchQuery
    })
  }
```